### PR TITLE
db seed small fixes

### PR DESCRIPTION
**Pull request by Jolg42**

### CODE REVIEW

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 1]
```diff
@@ -667,25 +667,14 @@ npx prisma db push --schema=/tmp/schema.prisma
...
 | Options             | Required | Description                             |
 | :------------------ | :------- | :-------------------------------------- |
 | `--help` / `--h`    | No       | Displays the help message               |
 
-#### Arguments
-
-| Argument   | Required | Description                                                                                                                                       | Default                                          |
-| :--------- | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------ | :----------------------------------------------- |
-| `--schema` | No       | Specifies the path to the desired schema.prisma file to be processed instead of the default path. Both absolute and relative paths are supported. | `./schema.prisma`<br /> `./prisma/schema.prisma` |
-
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/200-command-reference.mdx:688]: Not used anymore

**Comment by Jolg42** [comment]
We should add a link to https://deploy-preview-2183--prisma2-docs.netlify.app/docs/guides/database/seed-database I think

**Review by tomhoule — APPROVED** [review]

**Comment by Jolg42** [comment]
Screen Shot 2021-09-07 at 11 45 10

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 2]
```diff
@@ -667,25 +667,16 @@ npx prisma db push --schema=/tmp/schema.prisma
 
 ### <inlinecode>db seed</inlinecode>
 
-<Admonition type="warning">
-
-**MongoDB not supported** <br />
-`db push` uses the Migration Engine and does not currently support the [MongoDB connector](./../../concepts/database-connectors/mongodb).
+`db seed` changed from Preview to Generally Available (GA) in 3.0.1.
```
Comment (janpio) [inline content/400-reference/200-api-reference/200-command-reference.mdx:670]: Super uncommon wording for the docs. How do we usually do this @molebox?

Comment (Jolg42) [reply content/400-reference/200-api-reference/200-command-reference.mdx:670]: It's a copy paste from migrate one

**Pull request merged**